Dockerfile

```diff
@@ -17,13 +27,10 @@ RUN \
   git config --global pack.packSizeLimit 2047m && \
   git config --global pack.windowMemory 2047m
 
-# Docker 1.9.1 binary
-RUN \
-  wget https://get.docker.com/builds/Linux/x86_64/docker-1.9.1 && \
-  mv docker-1.9.1 /usr/bin/docker-1.9.1 && \
-  chmod +x /usr/bin/docker-1.9.1
-
 # Docker 17.09.0 binary
+# Used for cloud and enterprise before kubernetes in:
```

Comment:
Since it's not used, can we remove the section entirely?

Reply:
It is used, what do you mean by not used?

Reply:
We still use regular docker everywhere. We are still migrating cloud. At least that I am aware of.

Reply:
we can check with @gjsduarte we have to move forward, the projects that are still using the old image can stitch with it until the end of the year.

Reply:
That is never a good assumption.
I really don't understand why all this demands are appearing in my change in the base.
This is a PR to update the base, why is there a need to create all these problems in the middle?
If this was not used ok, but if it still is why not wait?

Reply:
I hope we can remove docker from this image soon